src/sql/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -543,7 +543,7 @@ class WelcomePage extends Disposable {
 		});
 	}
 
-	private async createListEntries(recents: (IRecentWorkspace | IRecentFolder)[], fileService): Promise<HTMLElement[]> {
+	private async createListEntries(recents: (IRecentWorkspace | IRecentFolder)[], fileService): Promise<any[]> {
```

Comment:
Any reason this changed to any[]?

Reply:
I was just experimenting with something, it's back to HTMLElement[]